pkg/reconciler/rolloutorchestrator/rolloutorchestrator.go

```diff
@@ -254,32 +259,38 @@ func UpdateSPAForRevDown(spa *v1.StagePodAutoscaler, revision *v1.TargetRevision
 	if !scaleUpReady {
 		return spa
 	}
+
+	if len(strings.TrimSpace(revision.RevisionName)) == 0 {
```

Comment:
Revision name can be empty ?

Reply:
It is not very possible to happen in real time, but I have a test case on it for safeguard reasons.